stack-test.yaml

```diff
@@ -0,0 +1,7 @@
+packages:
+- '.'
+- descriptor/
+- hprotoc/
+- protobuf-test-suite/
+extra-deps: []
+resolver: nightly-2018-05-21
```

Comment:
I think we should not have/need an extra stack.yaml for this. That way we could run the testsuite with travis on multiple GHC versions.